server/chat-plugins/helptickets.ts

```diff
@@ -17,40 +17,30 @@ interface TicketState {
 	created: number;
 	claimed: string | null;
 	ip: string;
+	resolution?: 'unknown' | 'dead' | 'unresolved' | 'resolved';
+	result: TicketResult | null;
+	claimQueue?: string[];
+	involvedStaff?: ID[];
+	createTime?: number;
+	emptyRoom?: boolean;
+	firstClaimTime?: number;
+	unclaimedTime?: number;
 }
```

Comment:
`TicketState` should all be `readonly`. There's a few obvious bugs due to writing to this.